rules/when-to-use-canva/rule.md

```diff
@@ -0,0 +1,35 @@
+---
+type: rule
+title: LinkedIn – Do you use Creator mode?
+uri: when-to-use-canva
+authors:
+  - title: Asher Paris
+    url: https://ssw.com.au/asher-paris
+related:
+  - use-memes-as-part-of-your-business-social-media-content
+  - rules-to-better-designers
+  - 
+created: 2023-12-13T13:42:55.753Z
+archivedreason: null
+guid: b9f9a660-5eab-4f6c-af8f-a9f35cd45df2
+---
+
+Canva is an intuitive online tool for creating images and crafty digital designs, particularly is effective for crafting engaging social media memes. However, it's not always the right tool for every design need. Which is why it's important to use Canva along aside design tools like Adobe, Figma and from time to time AI image generators.
+
+<!--endintro-->
+
+# Canva for Social Media: Best Practices
+
+For intricate and complex designs requiring advanced editing or branding designers, professional software like Figma, Adobe Photoshop or Illustrator is more appropriate. Do not use AI-generated images for social media content, as the current quality and control do not meet our standards (Note: This policy may be updated as AI imaging technology evolves).
```

Comment:
```suggestion
For intricate and complex designs requiring advanced editing or branding designers, professional software like Figma, Adobe Photoshop or Illustrator is more appropriate. Do not use AI-generated images for social media content, as the current quality is difficult to match to our professional style. (Note: This may be updated as AI imaging technology evolves).
```